### ISSUE TITLE
Not able to run in Ubuntu 20.04 for error related to PyQt5

### ISSUE THREAD

**Issue by rohan-paul** [adamerose/pandasgui#42]
Following [advice](https://github.com/adamerose/pandasgui/issues/34#issuecomment-701757849) from another closed issue from this repo tried this alternative way to run.

```python
import pandas as pd
from pandasgui import show
from PyQt5 import QtCore, QtGui, QtWidgets, QtWebEngineWidgets
from PyQt5.QtWebEngineWidgets import QWebEngineView
app = QtWidgets.QApplication([])
df = pd.DataFrame(([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])
show(df)
```

#### But getting below error

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-8c40c842eba1> in <module>
      1 import pandas as pd
----> 2 from pandasgui import show
      3 from PyQt5 import QtCore, QtGui, QtWidgets, QtWebEngineWidgets
      4 from PyQt5.QtWebEngineWidgets import QWebEngineView
      5 app = QtWidgets.QApplication([])

~/.local/lib/python3.8/site-packages/pandasgui/__init__.py in <module>
----> 1 from pandasgui.gui import show
      2 
      3 __all__ = ["show"]

~/.local/lib/python3.8/site-packages/pandasgui/gui.py in <module>
     12 from pandasgui.store import Store, PandasGuiDataFrame
     13 from pandasgui.utility import fix_ipython, fix_pyqt, get_logger, as_dict, delete_datasets
---> 14 from pandasgui.widgets.dataframe_explorer import DataFrameExplorer
     15 from pandasgui.widgets.find_toolbar import FindToolbar
     16 from pandasgui.widgets.json_viewer import JsonViewer

~/.local/lib/python3.8/site-packages/pandasgui/widgets/dataframe_explorer.py in <module>
      7 from pandasgui.utility import get_logger, nunique
      8 from pandasgui.widgets.dataframe_viewer import DataFrameViewer
----> 9 from pandasgui.widgets.grapher import Grapher
     10 from pandasgui.widgets.reshaper import Reshaper
     11 from pandasgui.widgets.filter_viewer import FilterViewer

~/.local/lib/python3.8/site-packages/pandasgui/widgets/grapher.py in <module>
     13 
     14 from pandasgui.utility import flatten_df, get_logger
---> 15 from pandasgui.widgets.plotly_viewer import PlotlyViewer
     16 from pandasgui.widgets.spinner import Spinner
     17 from pandasgui.widgets.dragger import Dragger, ColumnArg, Schema

~/.local/lib/python3.8/site-packages/pandasgui/widgets/plotly_viewer.py in <module>
     27         app.__init__(sys.argv)
     28     else:
---> 29         raise e
     30 
     31 

~/.local/lib/python3.8/site-packages/pandasgui/widgets/plotly_viewer.py in <module>
     15 # https://stackoverflow.com/a/57436077/3620725
     16 try:
---> 17     from PyQt5 import QtWebEngineWidgets
     18 except ImportError as e:
     19     if e.msg == "QtWebEngineWidgets must be imported before a QCoreApplication instance is created":

ImportError: /lib/x86_64-linux-gnu/libQt5Network.so.5: undefined symbol: _ZN15QIPAddressUtils8toStringER7QStringPh, version Qt_5
```

And I already have installed and upgraded PyQt5 as below

```
pip3 install --user --upgrade PyQt5

pip3 install --user --upgrade PyQt5-sip
```

**Comment by adamerose** [comment]
I think you might have a corrupted PyQt5 installation. Can you try just running `from PyQt5 import QtWebEngineWidgets`? Should be the same error, so it's not related to PandasGUI specifically. 

First thing I'd try to fix it is reinstall all PyQt packages:
`pip uninstall -y PyQt5 PyQtWebEngine PyQt5-sip`
`pip install PyQt5 PyQtWebEngine PyQt5-sip`

And if that doesn't work what's the output of `pip freeze | grep -i pyqt`?

**Comment by alessandrobessi** [comment]
@adamerose I have the same problem. The output of `pip freeze | grep -i pyqt` is
```
PyQt5==5.14.1
PyQt5-sip==12.8.1
PyQtWebEngine==5.14.0
```

Thanks

**Comment by rohan-paul** [comment]
So I finally resolved in Ubuntu 20.04 it as below.

1. Main reason for my case was, I had broken `pyqt5` - so I was not able to uninstall it and also NOT able to install a fresh version of `PyQt5`.

2. So finally had to manually delete  `PyQt5` + `PyQt5-sip` + `PyQtWebEngine`  respective installed directories from the below three locations.

```
/usr/lib/python3/dist-packages

/usr/local/lib/python3.8/dist-packages

/home/user_name/.local/lib/python3.8/site-packages
```

3. Then install them back with `pip install PyQt5 PyQtWebEngine PyQt5-sip`

4. Now, check the installed versions of those packages by running `pip freeze | grep -i pyqt`

Now got the upgraded and correct package version as below

```
PyQt5==5.15.1
PyQt5-sip==12.8.1
PyQtWebEngine==5.15.1
```

5. Then also had to uninstall `pandasgui` by running `pip uninstall pandasgui` and then reinstall it again.

And now `pandasgui`	is running fine.
